packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Status.Example.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { DefaultButton, getTheme, FontWeights, mergeStyleSets, DelayedRender, Callout } from 'office-ui-fabric-react';
+
+export interface IStatusCalloutExampleState {
+  isCalloutVisible?: boolean;
+}
+
+// Themed styles for the example.
+const theme = getTheme();
+const styles = mergeStyleSets({
+  buttonArea: {
+    verticalAlign: 'top',
+    display: 'inline-block',
+    textAlign: 'center',
+    margin: '0 100px',
+    minWidth: 130,
+    height: 32
+  },
+  callout: {
+    maxWidth: 300
+  },
+
+  subtext: [
+    theme.fonts.small,
+    {
+      margin: 0,
+      height: '100%',
+      padding: '24px 20px',
+      color: theme.palette.neutralPrimary,
+      fontWeight: FontWeights.semilight
+    }
+  ]
+});
+
+// Example code
+export class StatusCalloutExample extends React.Component<{}, IStatusCalloutExampleState> {
+  public state: IStatusCalloutExampleState = {
+    isCalloutVisible: false
+  };
+
+  private _menuButtonElement = React.createRef<HTMLDivElement>();
+
+  public render(): JSX.Element {
+    const { isCalloutVisible } = this.state;
+
+    return (
+      <>
+        <div className={styles.buttonArea} ref={this._menuButtonElement}>
+          <DefaultButton onClick={this._onShowMenuClicked} text={isCalloutVisible ? 'Hide StatusCallout' : 'Show StatusCallout'} />
+        </div>
+        {this.state.isCalloutVisible && (
+          <Callout
+            className={styles.callout}
+            target={this._menuButtonElement.current}
+            onDismiss={this._onCalloutDismiss}
+            role="status"
+            aria-live="assertive"
+          >
+            <DelayedRender>
```

Comment:
Is there a reason for using DelayedRender instead of Announced here? (Also for the fragment wrapping the p?)

Reply:
Fragment is required by DelayedRender. It's a bit picky.

Announced speaks the string but doesn't display it. To display an announced string, this is basically exactly what Announced does.

Reply:
Oh interesting, I forgot about that. Almost makes me wonder if we should have an Announced variant or prop which DOES show the content? I suspect it's a common enough scenario.

Reply:
Variant, or just base control. Would be good to abstract this usecase

Reply:
`<Announced show>`? And maybe reconsider the default in the Fluent version.

Reply:
I feel like this has come up for panel content or other cases too.